app/ui/legacy/src/main/res/layout/account_list_item.xml

```diff
@@ -1,49 +1,72 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<com.fsck.k9.view.DraggableFrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:background="?selectableItemBackground"
-    android:clickable="true"
-    android:focusable="true"
-    android:gravity="center_vertical"
-    android:orientation="horizontal"
-    android:paddingLeft="8dp">
-
-    <ImageView
-        android:layout_width="40dp"
-        android:layout_height="40dp"
-        android:contentDescription="@string/account_settings_action"
-        android:padding="8dp"
-        app:srcCompat="?attr/iconSettingsAccount" />
+    android:background="?android:attr/windowBackground"
+    android:foreground="?attr/selectableItemBackground"
+    android:stateListAnimator="@anim/draggable_state_list_anim"
+    tools:ignore="UnusedAttribute">
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="vertical"
-        android:paddingLeft="24dp"
-        android:paddingRight="16dp">
+        android:background="?android:attr/windowBackground"
+        android:clickable="true"
+        android:focusable="true"
+        android:foreground="?selectableItemBackground"
+        android:gravity="center_vertical"
+        android:orientation="horizontal"
+        android:paddingLeft="8dp">
 
-        <TextView
-            android:id="@+id/name"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:ellipsize="end"
-            android:paddingBottom="4dp"
-            android:paddingTop="8dp"
-            android:singleLine="true"
-            android:textAppearance="?android:attr/textAppearanceListItem"
-            tools:text="Personal" />
-
-        <TextView
-            android:id="@+id/email"
-            android:layout_width="wrap_content"
+        <ImageView
+            android:layout_width="40dp"
+            android:layout_height="40dp"
+            android:contentDescription="@string/account_settings_action"
+            android:padding="8dp"
+            app:srcCompat="?attr/iconSettingsAccount" />
+
+        <LinearLayout
+            android:layout_width="313dp"
```

Comment:
To make this view take up the available width use `android:layout_width="0dp"` and `android:layout_weight="1"`.